deployment/environment/memory/environment.go

```diff
@@ -49,6 +49,7 @@ type MemoryEnvironmentConfig struct {
 	Chains             int
 	SolChains          int
 	AptosChains        int
+	ZkChains           int
```

Comment:
I find it personally confusing that we have `ZkChains` but they use the same deployment funcs as EVM. Do we need a separate count for zk chains?

Reply:
We need a separate count for ZK because it uses different local blockchain (with the different VM). Note that I am using a different chain also from CTF https://github.com/smartcontractkit/chainlink/pull/17436/files#diff-08c2ac3acac29460f7518606245a0d69cfe8b98c731ec3196a2da07ac78e0819R34